mbbsd/email_challenge.cc

```diff
@@ -3,6 +3,12 @@ extern "C" {
 #include "daemons.h"
 }
 
+#ifdef USE_2FA
+# ifndef USE_VERIFYDB_ACCOUNT_RECOVERY
+#   error "USE_2FA requires USE_VERIFYDB_ACCOUNT_RECOVERY and USE_VERIFYDB"
```

Comment:
VERIFYDB is not strictly needed. You can always skip `LoadVerifyDbEmail()` and rely solely on user's listed email.